admin/release.py

```diff
@@ -406,7 +406,12 @@ def publish_homebrew_recipe(homebrew_repo_url, version, source_bucket,
 
     homebrew_repo.index.add([recipe])
     homebrew_repo.index.commit('Add recipe for Flocker version ' + version)
+
+    # Sometimes this raises an index error, and it seems to be a race
+    # condition. There should probably be a loop until push succeeds or
+    # whatever condition is necessary for it to succeed is met. FLOC-2043.
```

Comment:
I have linked to the above from the new issue, FLOC-2043 but won't modify it here as it is an unrelated change.